lib/std/zig/system.zig

```diff
@@ -110,6 +110,11 @@ pub const NativePaths = struct {
             try self.addLibDirFmt("/usr/lib{}", .{qual});
             try self.addLibDir("/usr/lib");
 
+            // Detect any native framework directories on MacOS
```

Comment:
Needs a check like `if (Target.current.isDarwin())`, we don't need those on other systems.